defaults-o2-dev-fairroot.sh

```diff
@@ -61,6 +61,18 @@ overrides:
     version: dev
     tag: dev
     source: https://github.com/FairRootGroup/FairRoot
+    requires:
+      - generators
+      - simulation
+      - ROOT
+      - ZeroMQ
+      - nanomsg
+      - boost
+      - protobuf
+      - DDS
```

Comment:
ZeroMQ, nanomsg, DDS, and googletest can be dropped from FairRoot recipe. (Protobuf too, if you do not build the examples).